kentikapi/internal/api_connection/rest_client.go

```diff
@@ -124,8 +161,12 @@ func (c *RestClient) Put(ctx context.Context,
 	return body, errorFromResponseStatus(response, string(body))
 }
 
-// Delete sends DELETE request to the API and returns raw response body.
+// Delete sends DELETE request to the API and returns raw response body
```

Comment:
Shouldn't there really be a period at the end of the sentence? Doesn't one of the linters insist on that?

Reply:
Done